Dockerfile

```diff
@@ -22,6 +22,11 @@ ARG LDFLAGS
 ARG GO_BUILD_GC_ARGS
 
 WORKDIR /workspace
+
+# Add kubectl tool to run helm hooks
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/${TARGETPLATFORM}/kubectl"
+RUN chmod +x ./kubectl
```

Comment:
Moved this up as it speeds up the build - this is easily cacheable when run before copying over files which may have changed.

Reply:
Will revert this and tackle it as part of #757